store/models/models.go

```diff
@@ -97,3 +97,24 @@ type Comment struct {
 func newComment(r *ReviewEvent, c *lookout.Comment) *Comment {
 	return &Comment{ID: kallax.NewULID(), ReviewEvent: r, Comment: *c}
 }
+
+// Organization is a persisted model for an Organization (e.g. a GitHub App
+// installation). It contains settings for a group of repositories.
+// The primary key should be (Provider,InternalID), but kallax does not support
+// composite or string keys
+type Organization struct {
+	kallax.Model `pk:"id"`
+	ID           kallax.ULID
+	Provider     string
+	InternalID   string
+	Config       string
```

Comment:
What are we storing in `Config`? It can be whatever thing depending on the Provider?

Reply:
Oh... I see... it stores the RAW version of the `.lookout.yml` for all repos under that org.
>_The default config will be merged with the `.lookout.yml` commited in the repository. In this case the one in the repository has priority over the organization default._
(from https://github.com/src-d/lookout/issues/401 )